src/components/BlogPost.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import '../BlogPost.css' //allows us to use the css from our other file
+
+//JavaScript Function
+const boldIt = (id:string) => {
+    document.getElementById(id)?.setAttribute("style", "font-weight: bold;")
+}
+
+const revert = (id:string) => {
+    document.getElementById(id)?.setAttribute("style", "font-weight: normal;")
+}
+
+function submitIt() {
+    alert('You have just submitted a post!');
+}
+
+function cancelIt() {
+    alert('You have just cancelled a post!');
+}
+
+//HTML/CSS
+const BlogPost: React.FC = () => {
+    return (
+        <div className="container">
+            <div className="blogPost">
+                <div>
+                    <h2 className="headTitle">Create A Blog Post</h2>
+                </div>
+                <form>
+                    <label id="title">Title</label><br />
```

Comment:
Can you put `<br />`s on a new line? It might not be very apparent that they're separate from the things they're being placed after.

Reply:
sure